### PR TITLE
Added CoGS for Dummies

### PR DESCRIPTION
**Pull request by lpulley**
Minor changes to the README. Will have to update the "Dummies" section as we add services. Also, @Timur2012, when cloning the repo and running from scratch, will the user need to run `docker-compose exec web python manage.py makemigrations` in addition to `migrate`, to create migrations from our custom models?

### CODE REVIEW

**Comment by mgale456** [comment]
Normally you'd run both commands for a new project. However, makemigrations just makes a file. Since anyone using this is downloading the repo, the migration file could already be made and just migrating works. I just tried tried it. You can probably run the command from that dockerfile. 

On that topic, we need to make an app instead of putting our models and stuff inside the project folder

**Comment by lpulley** [comment]
@mgale456 I'm always down to do things properly. I'll have to look into making things into apps in Django, since I'm not familiar, but you should definitely give it a go. You should create a branch off of `team-management` and convert the existing work into an app (just as soon as @Timur2012 pushes the branch here). I'll make sure you're a contributor. When you're done you can submit a pull request from your branch back onto `team-management`.

If you agree with my README changes you're welcome to approve this merge for me!

File: README.md [hunk 1]
```diff
@@ -16,18 +16,27 @@ For now, that's just [Django](https://www.djangoproject.com/) and [MongoDB](http
...
+The point of creating CoGS is to have a solution for the MRDC game server that is as portable, maintainable, and transparent as possible. We learned our lesson; we want CoGS to last for years and to be accessible to people who weren't here when we wrote it.
 
-In case this documentation is insufficient, or there are other problems with this system, you can contact [lpulley](https://lpulley.me/).
+In case this documentation is insufficient, or there are other problems with this system, you can contact [lpulley](https://github.com/lpulley/).
 
 ## Detailed Documentation
 
 ### Running and Stopping
 
 Run `docker-compose up -d` in the root of this repo to start all of the CoGS services in the background, or to update the containers if they are already running.
```
Comment (mgale456) [inline README.md]:
```suggestion
Run `docker-compose up -d` in the root of this repo to build images for every container and start all of the CoGS containers in the background, or to update the containers if they are already running.
```

File: README.md [hunk 2]
```diff
@@ -16,18 +16,27 @@ For now, that's just [Django](https://www.djangoproject.com/) and [MongoDB](http
...
 
-To stop the containers, run `docker-compose stop`. To restart them use `docker-compose start`. To stop the containers and delete them, use `docker-compose down` (note that while this does delete volumes, it does **not** delete host files, so for example the database data in `./data-db` will persist). The `down` subcommand is useful for starting from *almost* scratch, such that only data linked to the host filesystem persists, but every service is completely recycled.
+To stop the containers, run `docker-compose stop`. To restart them use `docker-compose start`. To stop the containers and delete them, use `docker-compose down` (note that while this does delete volume connections, it does **not** delete host files created by volumes, so for example the database data in `./data-db` will persist). The `down` subcommand is useful for starting from *almost* scratch, such that only data in volumes persists, but every service is completely recycled.
```
Comment (mgale456) [inline README.md]:
```suggestion
To stop the containers, run `docker-compose stop`. To restart them use `docker-compose start`. To stop the containers and delete them, use `docker-compose down` (note that while this does delete volume connections, it does **not** delete host files created by volumes or images, so for example the database data in `./data-db` will persist). The `down` subcommand is useful for starting from *almost* scratch, such that only data in volumes persists, but every service is completely recycled. Running `docker-compose up -d` will start them anew, but does not rebuild the images. To rebuild from the `docker-compose.yml` and `Dockerfile` you must remove the image with `docker rmi <imagename>` or `docker-compose --rmi all`
```

Comment (lpulley) [reply README.md]: Really good suggestion. Not sure why I didn't think to mention this. Thanks!

**Review by mgale456 — APPROVED** [review]
I like the extra clarifications for down and up and building images. COGS for dummies is also good and concise

**Review by mgale456 — APPROVED** [review]
trying to write an approving review

**Comment by lpulley** [comment]
Hahaha it says @mgale456 approved 1 hour ago, and has the check mark by your name, but still says it needs 1 approving review...hm. Let me know if you figure it out. I think we're just missing something silly.

**Comment by lpulley** [comment]
Oh @mgale456 make sure you accept my invitation to be a member of uiuc-mrdc! I think it's not counting your review because you're not officially in the organization.

**Comment by mgale456** [comment]
image
This is why I'm not in the organization. The invite got cancelled. Please reinvite.

**Comment by lpulley** [comment]
Weird, I didn't cancel it. Sent another just now.

**Comment by lpulley** [comment]
I'm going to leave the README branch open. Usually it's good to delete branches once their changes are in, but I intentionally made `readme` as an all-purpose branch for just `README.md`. Let's not do this with Django, though, that's a much bigger scope.

File: README.md [hunk 3]
```diff
@@ -16,18 +16,29 @@ For now, that's just [Django](https://www.djangoproject.com/) and [MongoDB](http
...
 
-To start with the database from scratch, remove `./data-db` after `docker-compose down`. When you start up again, the database will create a new `./data-db`. Note that if you do this, the database will be missing some vital default models from Django, so you must migrate the models to the database. In Django, this is done with `python manage.py migrate`. **However**, since Django is running inside the `web` service, you must use `docker-compose exec <service> <command>` or `docker exec <container> <command>`. In this case, that would be `docker-compose exec web python manage.py migrate`.
+The `down` subcommand is useful for starting from *almost* scratch, such that only data in volume folders persists, but every container is completely recycled. Running `docker-compose up -d` will start them anew, but does not rebuild images from `Dockerfile`s if they still exist. To rebuild images from `Dockerfile`s use `docker-compose build`. This will force a rebuild of images that are listed in `docker-compose.yml` with a `build:` directory.
```
Comment (lpulley) [inline README.md:31]: @mgale456 Not sure if you saw my slight change here, but I think you might find `docker-compose build` useful. I didn't know about it before but it does exactly what you're looking for, without having to remove images manually.

**Pull request merged**